pre-commit.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+
+if [ -z $AWX_IGNORE_BLACK ]
+then
+        black --check $(git diff --cached --name-only --diff-filter=AM | grep -E '\.py') || \
+        (echo 'To fix this, run `make black` to auto-format your code prior to commit, or set AWX_IGNORE_BLACK=1' && \
+        exit 1)
```

Comment:
should we mention `git add` needs to be run after the `make black`?

Reply:
probably not for `make black`, which is generally going to be something explicitly run by the user.